pennylane/utils.py

```diff
@@ -323,6 +326,22 @@ def circuit2():
             + ",".join(string_reps)
         )
 
+    if qml.tape_mode_active():
```

Comment:
Isn't this superfluous now, since tape mode is always active?

Reply:
Essentially. I left it in just to support the odd case of `qml.disable_tape()`.